package.json

```diff
@@ -6,16 +6,16 @@
   "license": "MIT",
   "scripts": {
     "start": "node server.js",
-    "start:dev": "npm run build && DEBUG=gov-starter-server* nodemon server.js | bunyan -o short",
-    "start-feature": "NOTIFY_ENABLED=false PORT=3007 NOMIS_AUTH_URL=http://localhost:9091/auth ELITE2API_ENDPOINT_URL=http://localhost:9091 DB_NAME=use-of-force-int DB_PORT=5433 NODE_ENV=development node $NODE_DEBUG_OPTION server.js | bunyan -o short",
-    "start-feature:dev": "NOTIFY_ENABLED=false PORT=3007 NOMIS_AUTH_URL=http://localhost:9091/auth ELITE2API_ENDPOINT_URL=http://localhost:9091 DB_NAME=use-of-force-int DB_PORT=5433 NODE_ENV=development nodemon $NODE_DEBUG_OPTION server.js | bunyan -o short",
+    "start:dev": "npm run build && DEBUG=gov-starter-server* DB_PORT=5433 nodemon server.js | bunyan -o short",
```

Comment:
Just checking this should be 5433 and not 5432?

Reply:
yes - we have two postgres DBs - one for test and one for dev so they don't interfere.
test ran on 5433 previously but we've had to swap it to 5432 and so dev has taken over 5433.
It's annoying for various reasons but easiest fix for now